src/sentry/conf/api_pagination_allowlist_do_not_modify.py

```diff
@@ -5,6 +5,7 @@
 """
 
 SENTRY_API_PAGINATION_ALLOWLIST_DO_NOT_MODIFY = {
+    "OrganizationUptimeCheckIndexEndpoint",
```

Comment:
this endpoint likely doesn't need to paginate. instead we can cap the number of uptime checks passed in, and the API already constrains the amount of time buckets